molo/core/content_import/tests/test_content_import.py

```diff
@@ -169,6 +168,22 @@ def test_import_sections_for_primary_language(self):
         self.assertEquals(ArticlePage.objects.all().count(), 46)
         self.assertEquals(FooterPage.objects.all().count(), 4)
 
+    def test_unknown_locale(self):
+        lang1 = eg_models.Localisation({'locale': 'eng_GB'})
+        lang2 = eg_models.Localisation({'locale': 'Zre_ZR'})
+
+        self.workspace.save(lang1, 'Added english language')
+        self.workspace.save(lang2, 'Added a language with unknown locale')
+
+        [cat_eng_1, cat_eng_2] = self.create_categories(
+            self.workspace, locale='eng_GB', count=2)
+        [cat_Zre_1, cat_Zre_2] = self.create_categories(
+            self.workspace, locale='Zre_ZR', count=2)
+
+        errors = ContentImportHelper(self.workspace).parse_locales()
+        self.assertEquals(errors[1], [
+            {'type': 'unknown_locale', 'details': {'locale': u'Zre_ZR'}}])
```

Comment:
Ah, sorry, read it the code wrong. Maybe it would be clearer if we did something like:

```py
locales, errors = ContentImportHelper(self.workspace).parse_locales()
self.assertEquals(errors, [...])
```